src/HotChocolate/Data/src/EntityFramework/Extensions/EntityFrameworkServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using HotChocolate.Execution.Configuration;
+using HotChocolate.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace HotChocolate.Data.Extensions
+{
+    public static class EntityFrameworkServiceCollectionExtensions
+    {
+        // todo: get rid of this
+        public static IServiceCollection AddDbContextInjection(
+            this IServiceCollection services)
+        {
+            return services.AddSingleton<IParameterExpressionBuilder, DbContextParameterExpressionBuilder>();
+        }
```

Comment:
I would like to get rid of this. Is there a better way to register the ParameterExpressionBuilder, without adding a reference to HotChocolate.Data.EntityFramework to HotChocolate.Types and registering the ParameterExpressionBuilder using the DefaultResolverCompiler?